config/environments/development.rb

```diff
@@ -2,7 +2,7 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Add an env variable instead of hard coding the value (for production purposes)

Reply:
Nervermind, This is the dev config, just do the same in config/environments/production.rb.